README.md

```diff
@@ -433,6 +433,8 @@ In flink 1.9, when creating a table, configuring the schema parameter is optiona
 | topicspattern | null | Multiple pulsar topics with more java regular matching | source |
 | partitiondiscoveryintervalmillis | -1 | Automatically discover increase and decrease topics, milliseconds. -1 means not open. | source |
 | clientcachesize | 5 | Number of cached pulsar clients | source, sink |
+| auth-params | null | pulsar clients auth| source, sink |
```

Comment:
@jianyun8023 Would you please open a new issue to add some code example for this feature? it would be much clear for user on how to use.